### PR TITLE
Remove unnecessary else clauses from various classes

### PR DESCRIPTION
**Pull request by ckelleyRH**
* Where appropriate check-and-cast and ternary operators were used
* Some logic was also simplified by removing Boolean literals

### CODE REVIEW

**Review by fmarco76 — APPROVED** [review]
LGTM

File: base/src/main/java/org/mozilla/jss/pkcs12/AuthenticatedSafes.java [hunk 1]
```diff
@@ -185,9 +185,8 @@ public SEQUENCE getSafeContentsAt(Password password, int index)
                 SEQUENCE.OF_Template seqt = new SEQUENCE.OF_Template(
                                                     SafeBag.getTemplate() );
                 return (SEQUENCE) ASN1Util.decode(seqt, os.toByteArray());
-              } else {
-                throw e;
               }
+            throw e;
```
Comment (edewata) [inline base/src/main/java/org/mozilla/jss/pkcs12/AuthenticatedSafes.java:189]: This code should be indented with 2 spaces to match the original code block. Ideally, the code block could be indented with 4 spaces, but it's up to you if you want to fix it or just leave it as is.

**Review by edewata — APPROVED** [review]
Just one minor comment, but otherwise it LGTM.

**Comment by ckelleyRH** [comment]
Thanks @edewata @fmarco76 !

**Pull request merged**